cluster/gce/gci/configure-helper.sh

```diff
@@ -1181,13 +1181,18 @@ function start-node-problem-detector {
   local -r km_config="${KUBE_HOME}/node-problem-detector/config/kernel-monitor.json"
   # TODO(random-liu): Handle this for alternative container runtime.
   local -r dm_config="${KUBE_HOME}/node-problem-detector/config/docker-monitor.json"
+  local -r custom_km_config="${KUBE_HOME}/node-problem-detector/config/kernel-monitor-counter.json"
```

Comment:
How is this file included in the release? Doesn't it need change to Bazel rules?

Reply:
It is part of the NPD release tarball.

Reply:
So it is added by the change in the NPD version.